src/common/services/WalletService.ts

```diff
@@ -187,7 +201,7 @@ export default abstract class WalletService {
       }
       while (this.hasSubscribers() && !this.areEnoughUnusedAddresses()) {
         // eslint-disable-next-line no-await-in-loop
-        await this.askForBalance(sessionId);
+        await this.askForBalance();
```

Comment:
We should probably also remove `sessionId` from the `startAskingForBalance` method on line 182.